doc/rbd/rbd-config-ref.rst

```diff
@@ -134,3 +134,54 @@ Read-ahead is automatically disabled if caching is disabled.
 :Type: 64-bit Integer
 :Required: No
 :Default: ``50 MiB``
+
+
+RBD Default Features
+========================
+
+RBD supports these features which can be specified via the command line when creating images or the default features can be specified via Ceph config file via 'rbd_default_features = <sum of feature numeric values>' or 'rbd_default_features = <comma-delimited list of CLI feature names>'
+
+``Layering``
+
+:Description: Layering enables you to use cloning.
+:Config numeric value: 1
```

Comment:
I'm not familiar enough w/ rst, but for some reason the value is getting moved down to the next line [1]

[1] http://docs.ceph.com/ceph-prs/24720/rbd/rbd-config-ref/#rbd-default-features

Reply:
I'll search if this can be resolved.

Reply:
Can we shorten this 'Config numeric value' keyword , its long as compared to the other ones . I think that's why the doc is not getting rendered properly.

Reply:
Sure -- perhaps try ```Internal value```?